src/Network/CorsBuilder.php

```diff
@@ -14,6 +14,8 @@
  */
 namespace Cake\Network;
 
+use Cake\Http\Response;
```

Comment:
in past `public function __construct(Response $response, $origin, $isSsl = false)` was used `Cake\Network\Response`
but i want to using within this method from `Cake\Http\Response`
but `Fatal error: Cannot use Cake\Http\Response as Response because the name is already in use in /var/www/cakephp-core/src/Network/CorsBuilder.php on line 17`
related to https://bugs.php.net/bug.php?id=66773
but i can not change method signature for BC ,what you think?
we can change these to bellow ?
```
use Cake\Http\Response as HttpResponse;

public function __construct(HttpResponse $response, $origin, $isSsl = false)
```